.github/blunderbuss.yml

```diff
@@ -13,220 +13,145 @@
-  - 'api: cloudkms'
-  - 'api: secretmanager'
-  - 'api: privateca'
-  - 'api: recaptchaenterprise'  
-  to:
-  - GoogleCloudPlatform/dee-infra
-- labels:
-  - 'api: vision'
-  to:
-  - GoogleCloudPlatform/ml-apis
-- labels:
-  - 'api: bigtable'
-  - 'api: datastore'
-  - 'api: firestore'
-  to:
-  - GoogleCloudPlatform/cloud-native-db-dpes
-- labels:
-  - 'api: asset'
-  - 'api: datacatalog'
-  - 'api: dataproc'
-  - 'api: clouderrorreporting'
-  - 'api: talent'
-  - 'api: vision'
-  to:
-  - GoogleCloudPlatform/python-samples-reviewers
-- labels:
-  - 'api: dlp'
-  to:
-  - GoogleCloudPlatform/googleapis-dlp
-- labels:
-  - 'api: functions'
-  to:
-  - GoogleCloudPlatform/functions-framework-google
+  # DEE teams
+  - labels:
+      - "api: appengine"
+      - "api: cloudbuild"
+      - "api: cloudscheduler"
+      - "api: cloudtasks"
+      - "api: containeranalysis"
+      - "api: eventarc"
+      - "api: functions"
+      - "api: run"
+      - "api: workflows"
+    to:
+      - GoogleCloudPlatform/torus-dpe
+  - labels:
+      - "api: asset"
+      - "api: batch"
+      - "api: compute"
+      - "api: cloudkms"
+      - "api: iam"
+      - "api: kms"
+      - "api: privateca"
+      - "api: recaptchaenterprise"
+      - "api: secretmanager"
+      - "api: securitycenter"
+    to:
+      - GoogleCloudPlatform/dee-infra
+  - labels:
+      - "api: clouderrorreporting"
+      - "api: cloudtrace"
+      - "api: container"
+      - "api: logging"
+      - "api: monitoring"
+      - "api: trace"
+    to:
+      - GoogleCloudPlatform/dee-platform-ops
+  - labels:
+      - "api: datacatalog"
+      - 'api: dataflow'
+      - "api: datalabeling"
+      - "api: dataproc"
+      - "api: dialogflow"
+      - "api: enterpriseknowledgegraph"
+      - "api: language"
+      - "api: ml"
+      - "api: notebooks"
+      - "api: optimization"
+      - "api: people-and-planet-ai"
+      - "api: speech"
+      - "api: talent"
+      - "api: texttospeech"
+      - "api: translate"
+      - "api: vision"
+    to:
+      - GoogleCloudPlatform/dee-data-ai
+
+  # AppEco teams
+  - labels:
+      - "api: cloudsql"
+    to:
+      - GoogleCloudPlatform/infra-db-dpes
+  - labels:
+      - "api: bigtable"
+      - "api: datastore"
+      - "api: firestore"
+    to:
+      - GoogleCloudPlatform/cloud-native-db-dpes
+  - labels:
+      - "api: spanner"
+    to:
+      - GoogleCloudPlatform/api-spanner-python
+  - labels:
+      - "api: storage"
+    to:
+      - GoogleCloudPlatform/cloud-storage-dpes
+  - labels:
+      - "api: pubsub"
+      - "api: pubsublite"
+    to:
+      - GoogleCloudPlatform/api-pubsub-and-pubsublite
+
+  # AppEco individuals
+  - labels:
+      - "api: bigquery"
+    to:
+      - shollyman
+  - labels:
+      - "api: composer"
+    to:
+      - leahecole
+      - rachael-ds
+      - rafalbiegacz
+  - labels:
+      - "api: datascienceonramp"
+    to:
+      - leahecole
+      - bradmiro
+  - labels:
+      - "api: contentwarehouse"
```

Comment:
`contentwarehouse` and `documentai` were previously assigned to `data-ai`. Is there a reason for assigning them to an individual?

Reply:
Document AI ownership is in progress. How about we use this team for now? hhttps://github.com/orgs/GoogleCloudPlatform/teams/document-ai-samples-contributors/members

We can then consider a `document-ai-samples-owners` if needed in the future.

Reply:
SGTM, please update.